README.rst

```diff
@@ -14,46 +14,55 @@ risk!
 Installing
 ==========
 
+|build status|
+
 Prerequisites
 -------------
 
 * Fortran 2003 compliant compiler
 
+* C compiler (for automatic toolchain selection)
```

Comment:
Why do we need that?

Reply:
We actually don't, it was an artifact of the automatic compiler toolchain selection.